.vscode/settings.json

```
@@ -1,3 +1,4 @@
 {
-    "omnisharp.autoStart": false
+    "omnisharp.autoStart": false,
+    "restructuredtext.confPath": "${workspaceFolder}/docs/codeql"
```

```suggestion
    "omnisharp.autoStart": false
```
Is it okay to revert this rST config change? It might cause difficulties if people have a different workspace root 🙂

---

Oops, did mean to commit that!